worksheet_D/oxo.py

```diff
@@ -0,0 +1,222 @@
+class OxoBoard:
+    def __init__(self):
+        """
+        For nxn I would have to choose a different
+        naming scheme something like ZeroZero,OneZero representing x and y
```

Comment:
Good explanation of how the code would be expanded for n*n. However the proposed changes would be a lot of work.